iot_config/migrations/3_routes.sql

```diff
@@ -0,0 +1,36 @@
+create table routes (
+    id uuid primary key not null default uuid_generate_v1mc(),
+    oui bigint not null references organizations(oui) on delete cascade,
+    net_id bigint not null,
+    max_copies int not null,
+    server_host text not null,
+    server_port int not null,
+    server_protocol_opts json not null,
+
+    inserted_at timestamptz not null default now(),
+    updated_at timestamptz not null default now()
+);
+
+select trigger_updated_at('routes');
+
+create table route_eui_pairs (
+    route_id uuid not null references routes(id) on delete cascade,
+    app_eui bigint not null,
+    dev_eui bigint not null,
+
+    inserted_at timestamptz not null default now(),
+    updated_at timestamptz not null default now()
+);
+
+select trigger_updated_at('route_eui_pairs');
+
+create table route_devaddr_ranges (
+    route_id uuid not null references routes(id) on delete cascade,
+    start_nwk_addr int not null,
+    end_nwk_addr int not null,
```

Comment:
Are these only the `nwk_addr` bits of a DevAddr?

Reply:
crap; no, that should be bigint and encompass the full integer of the devaddr; it looks like they originally started out as bigints in the _original_ route migrations for the elixir service and were downgraded to integers for some reason and that's what i carried over here